tools/allocatable-diff/go.mod

```diff
@@ -1,44 +1,46 @@
 module github.com/aws/karpenter-provider-aws/tools/allocatable-diff
 
-go 1.22
+go 1.22.4
 
 require (
-	github.com/aws/karpenter-provider-aws v0.33.1-0.20231206223517-f73ccfa65419
+	github.com/aws/karpenter-provider-aws v0.37.1-0.20240607201537-69a807081c7f
```

Comment:
Why are we updating this?

Reply:
To make my life easier with the `go.work` file :P